quic/s2n-quic/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "s2n-quic"
-version = "1.6.1"
+version = "1.6.2"
```

Comment:
wondering why this is a patch update rather than a minor bump?

Reply:
Moving to `1.7.1`